src/renderer/react/shapes/module.js

```diff
@@ -0,0 +1,11 @@
+import PropTypes from 'prop-types';
+
+export const moduleShape = {
+  icon: PropTypes.node,
+  manifest: PropTypes.shape({
+    name: PropTypes.string.isRequired,
+    displayName: PropTypes.string,
```

Comment:
I don't know, I wanted to give the possibility to add a displayName since the name is the technical id

If one day we move modules elsewhere we could imagine having:
```
name: '@developer-vault/notes',
displayName: 'Notes',
```